### PR TITLE
structured names for structural to functional test registration

### PR DESCRIPTION
**Pull request by damaggu**

### CODE REVIEW

File: samri/pipelines/nipype_based/test_reg.py [hunk 1]
```diff
@@ -199,7 +200,7 @@ def structural_to_functional_per_participant_test(subjects_sessions,
 			_n4.inputs.shrink_factor = 2
 			_n4.inputs.n_iterations = [500,500,500,500]
 			_n4.inputs.convergence_threshold = 1e-14
-			_n4.inputs.output_image = 'ss__n4_{}_ofM{}.nii.gz'.format(*subject_session.values())
+			_n4.inputs.output_image = '{}_{}_1_biasCorrection_slow.nii.gz'.format(*subject_session.values())
```
Comment (TheChymera) [inline samri/pipelines/nipype_based/test_reg.py]: @damaggu  the meaningful difference between this and the previous bias correction is not the speed, but rather that the result of the previous one (`n4_res`) is meant to be further processed and used for the registration, whereas the result of this bias correction is just used to apply a threshold based on which to calculate a brain mask.

Comment (TheChymera) [inline samri/pipelines/nipype_based/test_reg.py]: @damaggu the meaningful difference between this and the previous bias correction is not the speed, but rather that the result of the previous one (n4_res) is meant to be further processed and used for the registration, whereas the result of this bias correction is just used to apply a threshold based on which to calculate a brain mask.

Comment (TheChymera) [reply samri/pipelines/nipype_based/test_reg.py]: Can you please update the names accordingly?

Comment (damaggu) [reply samri/pipelines/nipype_based/test_reg.py]: I changed accordingly. Also, why is one of the bias corrections more suited for brain mask and the other for further processing and registration?

**Pull request merged**